components/game/TurnStatusMessage.tsx

```diff
@@ -13,9 +21,38 @@ export const YourTurnStatusMessage = () => {
 export const OpponentTurnStatusMessage = () => {
   return (
     <Center>
-      <Alert title="It is your opponents turn!" radius="xl" color="yellow">
+      <Alert
+        mt="sm"
+        title="It is your opponents turn!"
+        radius="md"
+        color="orange"
+        style={{ width: '100%', border: '1px solid #FFA94D' }}
+      >
         Sit back and relax or plan your next move!
       </Alert>
     </Center>
   )
 }
+
+export const EndTurnStatusMessage = () => {
+  const gameContext = useContext(GameContext)
+  return (
+    <Center>
+      <Alert
+        mt="sm"
+        title="The game is over"
+        radius="md"
+        color="dark"
+        style={{ width: '100%', border: '1px solid #909296' }}
+      >
+        {gameContext?.userPoints &&
+          gameContext.opponentPoints &&
+          (gameContext?.userPoints > gameContext?.opponentPoints
+            ? 'You won the game'
+            : gameContext?.userPoints < gameContext?.opponentPoints
+            ? `${gameContext?.opponentName} won the game`
+            : 'The game has no winner')}
```

Comment:
Nested ternaries som dette kan være vanskelig å debugge, men om du er sikker på at dette funker så er det c00lio for meg

Reply:
Ser den! De to første linjene må være der fordi poengene kan være undefined 😅 Ellers syns jeg et er ganske straight forward! Har testet det, og fungerer fint 😇